### PR TITLE
conf-unwind: Add support for fedora-base distributions

### PR DESCRIPTION
**Pull request by kit-ty-kate**

### CODE REVIEW

**Comment by mseri** [comment]
Do we need to do this on all conf- packages?

File: packages/conf-unwind/conf-unwind.0/opam [hunk 1]
```diff
@@ -10,7 +10,7 @@ depexts: [
   ["libunwind-dev"] {os-family = "debian" | os-family = "ubuntu"}
   ["libunwind-devel"] {os-distribution = "centos"}
   ["libunwind-devel"] {os-distribution = "rhel"}
-  ["libunwind-devel"] {os-distribution = "fedora"}
+  ["libunwind-devel"] {os-distribution = "fedora" | os-family = "fedora"}
```
Comment (mseri) [inline packages/conf-unwind/conf-unwind.0/opam:13]:
```suggestion
  ["libunwind-devel"] {os-family = "fedora"}
```
Why not this? Can it have fedora distribution but not family?

Comment (kit-ty-kate) [reply packages/conf-unwind/conf-unwind.0/opam:13]: mmh, good question, I thought Fedora itself had `ID_LIKE = rhel` or something but it turns out they do not (maybe they had in the past?). So I guess this should be safe to do.

Comment (mseri) [reply packages/conf-unwind/conf-unwind.0/opam:13]: I'll keep it as is, can be revisited in the future

**Pull request merged**